INSTALL.md

```diff
@@ -1,10 +1,16 @@
-## Installing the CircleCI Runner
+# Installing the CircleCI Runner
 
-Currently only Ubuntu 18.04 running on an AMD64 or ARM64 architecture is officially supported. Running in a container is not supported at this time.
+## Supported platforms
 
-### Prerequisites
+At present, the supported platforms are:
+* Ubuntu 18.04 running on an AMD64 or ARM64 architecture
+* macOS Catalina (10.15.6+) or above, running on and AMD64 architecture.
 
-#### Authentication
+Running in a container is not supported at this time.
```

Comment:
@appplemac in a recent GTM meeting there was some discussion about tiers of support - is that something we can start adding into the docs?

I'd like to put Docker in a "try it if you want" category